roles/openshift_master/tasks/main.yml

```diff
@@ -59,25 +54,32 @@
     openshift_registry_url: "docker-registry.ops.rhcloud.com/openshift3_beta/ose-${component}:${version}"
   when: openshift.common.deployment_type == 'online' and openshift_registry_url is not defined
 
+# TODO: Need to get a flag added for volumes path, i think it'll get put in
 - name: Create master config
   command: >
-    /usr/bin/openshift start master --write-config
-    --config={{ openshift_master_config }}
+    /usr/bin/openshift start master
+    --write-config={{ openshift_master_config_dir }}
```

Comment:
Indeed, I was helping troubleshoot a user issue and thought it was expecting a filename.